src/platform/assets/composables/useUploadModelWizard.ts

```diff
@@ -62,6 +62,15 @@ export function useUploadModelWizard(modelTypes: Ref<ModelTypeOption[]>) {
   async function fetchMetadata() {
     if (!canFetchMetadata.value) return
 
+    // Clean and normalize URL
+    let cleanedUrl = wizardData.value.url.replace(/\s+/g, '')
```

Comment:
Some URLs have spaces in them that would normally be encoded as `%20` but this would just remove them.
I think you might want to do a `.trim()` and then [`encodeURI`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI).